source/iterators.tex

```diff
@@ -4043,14 +4051,30 @@
 
 \indexlibrarymember{base}{move_iterator}%
 \begin{itemdecl}
-constexpr Iterator base() const;
+constexpr Iterator base() const &;
 \end{itemdecl}
 
 \begin{itemdescr}
+\pnum
+\constraints \tcode{Iterator} satisfies \tcode{CopyConstructible}.
```

Comment:
I think this should be `\libconcept{CopyConstructible}`.

Reply:
Ok, changed to "the \oldconcept{CopyConstructible} requirements".